editions/multiwikiserver/tiddlywiki.info

```diff
@@ -11,6 +11,18 @@
 		"tiddlywiki/snowwhite"
 	],
 	"build": {
+		"--mws-list-users": [
```

Comment:
I think it would be confusing to have a named build that matches a built-in command. Perhaps this one isn't necessary in any case?